### DISCUSSION TITLE
Mac端tg连接有问题，申请退款

### DISCUSSION THREAD

**Discussion by daerontang** [getlantern/lantern-forum-cn#1072] [Q&A]
@Mike777Khoo 
购买了一年，发现在Mac端的tg连接存在问题，官方没开启申请退款通道，麻烦帮忙退款下，支付收据如下。
账户：598872123@qq.com
账单详情
武汉雄少优科技有限公司 -349.00
交易成功
支付时间 2024-12-07 15:41:03
付款方式 招商银行储蓄卡(3427)
商品说明 订单疑问q1053189871
收款方全称 武汉雄少优科技有限公司
订单号 2024120722001416401447716565
商家订单号20241207154018358779369
![微信图片_20241208001650](https://github.com/user-attachments/assets/d84c6709-ee85-4edd-8f21-0306886c34a8)
上学期间用了三年蓝灯，工作三年来再次用发现体验不及当年，希望后面能搞好吧。

**Comment by MaxMKMax** [comment]
@daerontang 您好. 可以的, 我帮您提交问题上去处理. 谢谢您支持蓝灯!

但是退款之前请您帮个忙. 需要您提交应用内报告. 我们这里才可以退款. 因为我们也想找出问题在哪里.

您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@daerontang 退款之前请您帮个忙. 需要您提交应用内报告. 我们这里才可以退款. 因为我们也想找出问题在哪里.

**Reply by daerontang** [reply]
我已经在软件内提交了报告，但一天了一直没有收到你提到的邮件以及8位数号码，下面该怎么搞？
<img width="1363" alt="Pasted Graphic" src="https://github.com/user-attachments/assets/244ad586-d37c-4555-8776-f9d21b1adb4a">
